cmd/genji/shell/shell.go

```diff
@@ -32,7 +32,7 @@ type Shell struct {
 	livePrefix string
 	multiLine  bool
 
-	history        []string
```

Comment:
I think if you rebase with master this is already formatted correctly

Reply:
I will see.